python/ requirements.txt

```diff
@@ -0,0 +1,5 @@
+sqlalchemy
```

Comment:
File name - `requirements.txt` seems to be prefixed with extra space, can we please remove it?

Reply:
Done. Thanks :)